project-management-application-web/src/app/auth/components/login/login.component.ts

```diff
@@ -3,13 +3,22 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
-  styleUrls: ['./login.component.css']
+  styleUrls: ['./login.component.css','./login.themes.component.css'],
 })
 export class LoginComponent implements OnInit {
+  text: String ='password';
```

Comment:
better go with a `boolean`  something like `showPassword` and you just toggle the value